package.json

```diff
@@ -1,13 +1,13 @@
 {
-    "name": "poker-player-javascript",
```

Comment:
You probably have a default setting to use 2 spaces for indentation (I do too) but it should be overridable by the project's default. [Atom also supports editorconfig](https://github.com/sindresorhus/atom-editorconfig#readme).